KelynPNjeri/Zen
# #167551829 Setup Folder Structure.

#### What does this PR do?
- This  PR contains the folder structure for the Zen Web App.

#### Description of Task to be completed?
- [x] Setup Folder Structure for Zen.

#### How should this be manually tested?
- Clone this repository.
- Checkout to the ch-setup-folder-structure-167551829 branch using `git checkout ch-setup-folder-structure-167551829`
- Observe the folder structure using the `tree` command.

#### Any background context you want to provide?
- None.

#### What are the relevant pivotal tracker stories?
[#167551829](https://www.pivotaltracker.com/story/show/167551829)

#### Screenshots (if appropriate)
- N/A
